src/aere_repl.erl

```diff
@@ -141,6 +142,32 @@ apply_command(State, type, I) ->
     TypeStr = aeso_ast_infer_types:pp_type("", Type),
     TypeStrClean = re:replace(TypeStr, ?TYPE_CONTAINER ++ "[0-9]*\\.", "", [global, {return, list}]),
     {aere_msg:output(TypeStrClean), State};
+apply_command(State = #repl_state{contract_state = {_, OldType, _}}, state, I) ->
+    [Expr] = aere_sophia:parse_body(I),
```

Comment:
I think you would rather parse expr, not body.

Reply:
And if you stick to body, drop assumption of single-element list.

Reply:
Used `parse_body/1` because there's no `parse_expr/1` and `expr/0` is not exported from `aeso_parser` acca9aa8fd3ae2c03fbc72ef6e66cae0f7d2f5b3